docs/resources/connector_gcp_vpc.md

```diff
@@ -26,6 +34,66 @@ resource "alkira_connector_gcp_vpc" "gcp_vpc1" {
 }
 ```
 
+You could adjust routing options on the VPC by using
+`gcp_routing` and `prefix_list_ids` along with resource
+`alkira_policy_prefix_list`:
+
+```terraform
```

Comment:
For examples, you will need to add new one in `examples` and add multi examples in the template file. Refer to connector-aws-vpc.